npm.mk

```diff
@@ -0,0 +1,23 @@
+npm = docker run -it --rm \
```

Comment:
L'utilisation de docker pour npm est pratique, mais peut poser problème dans le cas de l'utilisation d'un packager conteneurisé ([docker in docker](http://jpetazzo.github.io/2015/09/03/do-not-use-docker-in-docker-for-ci/))